project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=1.0.0-RC2
```

Comment:
Yes, it doesn't seem to be on brew yet and I knew this was blocked anyway for now, hence the laziness :)
I'll make sure everything is up to date by the time I finish